sdk/Eversign/Signer.php

```diff
@@ -179,7 +179,7 @@ public function getSigned() {
         return !!$this->signed;
     }
 
-    public function getSigned_timestamp() {
+    public function getSignedTimestamp() {
```

Comment:
getSignedTimestamp would be more consistent than `getSigned_timestamp`.
However, this is public function and this was released since a long time.

Meaning, changing it right now would introduce a breaking change. It might break other integrations. 
On the other hand this is not a big enough change to make a major release.

So, in short, we can not change this now. Please revert this part of the change.